pkg/repositories/models/reservation.go

```diff
@@ -0,0 +1,19 @@
+package models
+
+import "time"
+
+type ReservationKey struct {
+	DatasetProject string `gorm:"primary_key"`
```

Comment:
can you delete a dataset or tag? should this be relational?

Reply:
There is no datacatalog API for deleting a dataset/tag.